repos/c2sm/packages/infero/package.py

```diff
@@ -29,8 +29,8 @@ class Infero(CMakePackage):
     variant('tf_c', description='Enable tensorflow-c backend', default=False)
     variant('onnx', description='Enable ONNX backend', default=False)
 
-    depends_on('eckit@1.20.0:')
-    depends_on('fckit')
+    depends_on('eckit@1.20.2')
+    depends_on('fckit@0.9.0')
```

Comment:
This pins 1 version of eckit and 1 version of fckit. That's quite a narrow range.

Reply:
Yes I know, but there are some API-changes that require me to stick to that versions.
Also infero is only supported for one specific tag.
And since it is part of a research project that will end soon I   don't want to invest further resources here.